drivers/volume/aws/aws.go

```diff
@@ -122,11 +122,6 @@ func (a *aws) OwnsPVCForBackup(
 		// If user has forced the backupType in config map, default to generic always
 		return false
 	}
-	// For AWS volume and backuplocation type is NFS, we will not own.
-	// It will default to kdmp
-	if blType == storkapi.BackupLocationNFS {
```

Comment:
We may need to remove this argument blType from  function call altogether.. I guess it is no more used anywhere now ?